app/templates/_package.json

```diff
@@ -15,7 +15,8 @@
     "gulp-imagemin": "^2.2.1",
     "gulp-load-plugins": "^0.10.0",
     "gulp-minify-css": "^1.1.1",
-    "gulp-minify-html": "^1.0.0",<% if (includeSass) { %>
+    "gulp-minify-html": "^1.0.0",
+    "gulp-rev-all": "^0.8.0",<% if (includeSass) { %>
```

Comment:
I'm sorry, but `gulp-rev-all` is [blacklisted](https://github.com/gulpjs/plugins/blob/master/src/blackList.json#L70).

> "gulp-rev-all": "duplicate of gulp-rev",

[gulp-rev](https://github.com/sindresorhus/gulp-rev) should be able to get the job done.